### DISCUSSION TITLE
流動性的放置選擇

### DISCUSSION THREAD

**Discussion by yurenju** [lun-dao/LunDAO#63] [General]
因為我一直很想要先放一部份流動性到 Uniswap，所以又再次開了主題來討論這個 🤣

我們在 #49 已經有討論了一些流動性的話題，那個時候是提到先不要提供流動性，我想大家主要的考量是擔心價格會跌得太多（或是有其他考量可能我沒意會到）。但是我覺得流動性的提供還是滿重要的，因為一來 RFA 會需要其他人購買，二來是作者會需要可以把兌現。

如果在擔心崩盤的前提下，我想要提一個新的流動性方案，就是我們在 Uniswap v3 上面放一個 range order。比如說以前 Uniswap v2 我們放 1 ETH : 30,000 LUN，以 ETH 3000 USD 來算就是 1 LUN = 0.1 USD，但如果領到 LUN 代幣的人一直賣，價格就會一直跌。

但是如果放 Uniswap v3 的 range order，我們可以放一個 range 是  0.1 USD < 1 LUN < 10 USD，這樣的話如果超出這個範圍就會沒有流動性可以交易。這樣的話如果真的太跌就會變成沒辦法交易。

假設我們這麼做：
- 拿 3 ETH : 90,000 LUN 來提供流動性
- 設定 range 是 0.00003 ETH ~ 0.003 ETH (以 1 ETH = 3000 USD 來算大約是 0.1 USD ~ 10 USD)

這樣的話就可以達成前面說的效果，也就是價格太低的話就沒辦法使用 LunDAO 的流動性。想知道 @iisaint 跟 @hydai 覺得這樣如何？

**Comment by iisaint** [comment]
這樣很好，可以有限度地進行市場測試，超讚
